plugins/generator-1.21.8/neoforge-1.21.8/procedures/gui_set_items.java.ftl

```diff
@@ -1,7 +1,10 @@
 <#include "mcitems.ftl">
-if(${input$entity} instanceof Player _player && _player.containerMenu instanceof ${JavaModName}Menus.MenuAccessor _menu) {
-	ItemStack _setstack = ${mappedMCItemToItemStackCode(input$item, 1)}.copy();
-	_setstack.setCount(${opt.toInt(input$amount)});
-	_menu.getSlots().get(${opt.toInt(input$slotid)}).set(_setstack);
+<@head>if(${input$entity} instanceof Player _player) {
+  	if (_player.containerMenu instanceof ${JavaModName}Menus.MenuAccessor _menu) {</@head>

```

Comment:
This head is still too specific. Currently, it will reuse changes only if one sets same slot to same item multiple times, which makes little sense